default.nix

```
@@ -1,11 +1,6 @@
-let
-  lock = builtins.fromJSON (builtins.readFile ./flake.lock);
-  locked = lock.nodes.flake-compat.locked;
-  compat = fetchTarball {
-    url = "https://github.com/${locked.owner}/${locked.repo}/archive/${locked.rev}.tar.gz";
-    sha256 = locked.narHash;
-  };
+{ pkgs }:
```

Given that this file is for explicitly avoiding using flakes, maybe it's worth defaulting to importing`<nixpkgs>` for convenience

```suggestion
{ pkgs ? import <nixpkgs> {} }:
```